app/util.js

```diff
@@ -108,19 +108,19 @@ const searchword = location => {
         if (word && url) {
             const q = location.replace(`${word} `, "")
             if (q && location.startsWith(`${word} `)) {
-                const queries = q.split(",").map(item => item.trim())
+                const queries = q.split(",")
                 let urlString = url
                 let counter = 1
                 const patternMatches = (urlString.match(/%s/g) || []).length
                 while (urlString.includes("%s") && counter < patternMatches) {
-                    if (0 in queries) {
-                        urlString = urlString.replace(/%s/, encodeURIComponent(queries.shift()))
+                    if (queries[0]) {
+                        urlString = urlString.replace(/%s/, encodeURIComponent(queries.shift().trim()))
```

Comment:
This is trimmed here to allow for this kind of use case:
`ghrepo Jelmerro, Vieb`
This will get changed to:
`github.com/Jelmerro/Vieb/`